test/tic_tac_toe/ai_test.clj

```diff
@@ -192,4 +194,49 @@
   (let [player  (->AiPlayer "o")
         game { :board [["_" "_" "_" ]["_" "_" "_" ]["_" "_" "_" ]] :ai-marker "o" :player-marker "x"}]
     (testing "creates defrecord of player protocol"
-      (is (= (assoc game :board [["o" "_" "_"] ["_" "_" "_"] ["_" "_" "_"]]) (next-move player game))))))
\ No newline at end of file
+      (is (= (assoc game :board [["o" "_" "_"] ["_" "_" "_"] ["_" "_" "_"]]) (next-move player game))))))
+
+(defn win-or-draw? [player-marker winner-state]
+ (or (= player-marker winner-state) (= player-marker "its a draw")))
+
+(defn correct-ai-choice? [game]
+  (let[winner (winner? (:board game))]
+     (cond
+      (= winner (:ai-marker game)) true
+      (= winner "its a draw") true
+      (= winner nil) true
+      :else false )))
+
+(declare check-every-possible-gamestate)
+
+(defn check-gamestate [gamestate]
```

Comment:
I would recommend either pulling these helper functions to the top of the file, or extracting them to a helper namespace. You want to group your tests and helpers together so when people scan through the code they can find the pieces they want easily.